app/experimenter/nimbus-ui/src/components/GQLErrorAlert/mocks.tsx

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const NETWORK_SERVER_ERROR = {
+  graphQLErrors: [],
+  networkError: {
+    name: "ServerError",
+    response: {},
+    statusCode: 400,
+    result: {
+      errors: [
+        {
+          message: 'Cannot query field "WOOT" on type "NimbusExperimentType".',
+          locations: [
+            {
+              line: 3,
+              column: 5,
+            },
+          ],
+        },
+      ],
+    },
+  },
+  message: "Response not successful: Received status code 400",
+};
+
+export const NETWORK_SERVER_PARSE_ERROR = {
+  graphQLErrors: [],
+  networkError: {
+    name: "ServerParseError",
+    response: {},
+    statusCode: 404,
+    bodyText:
```

Comment:
I forced each of these errors to happen and copied them exactly as-is to have mocks matching exactly what we should receive in practice. Thought about taking a chunk of this one out, but🤷‍♀️